application/client.core/src/app/environment/layout/contextmenu/component.ts

```diff
@@ -37,11 +37,13 @@ export class LayoutContextMenuComponent implements OnDestroy, AfterViewInit {
         Object.keys(this._subscriptions).forEach((key: string) => {
             this._subscriptions[key].unsubscribe();
         });
-        this._unsubscribeToWinEvents();
+        ContextMenuService.unsubscribeToWinEvents(EEventType.keydown, this._onWindowKeyDown.bind(this));
+        ContextMenuService.unsubscribeToWinEvents(EEventType.mousedown, this._onWindowMouseDown.bind(this));
     }
 
     public ngAfterViewInit() {
-        this._subscribeToWinEvents();
+        ContextMenuService.subscribeToWinEvents(EEventType.keydown, this._onWindowKeyDown.bind(this));
+        ContextMenuService.subscribeToWinEvents(EEventType.mousedown, this._onWindowMouseDown.bind(this));
     }
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Function/bind

> The bind() method creates a new function that, when called, has its this keyword set to the provided value, with a given sequence of arguments preceding any provided when the new function is called.

It means you will not unsubscribe events, because you push **new function** there, but not existed handler